python/default_crab_config.py

```diff
@@ -14,13 +14,9 @@ def create_config(is_mc):
     config.General.transferLogs = True
 
     config.JobType.pluginName = 'Analysis'
-    config.JobType.psetName = '../python/dummy_pset.py'
-    config.JobType.scriptExe = '../bin/runFrameworkOnGrid.sh'
+    config.JobType.psetName = '../../Framework/test/TestConfiguration.py'
     config.JobType.sendPythonFolder = True
```

Comment:
I guess this one can be turned off since we don't use python anymore for the framework. It does not hurt to let it to True expect increasing a bit the task transfer time because of a bigger sandbox file